extensions/amp-story/0.1/system-layer.js

```diff
@@ -213,8 +229,12 @@ export class SystemLayer {
     this.root_.addEventListener('click', e => {
       const target = dev().assertElement(e.target);
 
-      if (target.matches(`.${FULLSCREEN_CLASS}, .${FULLSCREEN_CLASS} *`)) {
+      if (target.matches(
+          `.${EXIT_FULLSCREEN_CLASS}, .${EXIT_FULLSCREEN_CLASS} *`)) {
```

Comment:
Not your fault but we should be using `dom.js#matches` helper. Native `matcher` has inconsistent browser support.

I think using `closest` or `closestBySelector` is cleaner though.

Reply:
Oops, merged already. Created a GFI for this: #11816